articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -74,15 +74,26 @@ The following parameters are used by the App Configuration Push task:
         "TestApp:Settings:Message": "Message data"
     }
     ```
+- **File Content Profile**: The Configuration File's [content profile](https://docs.microsoft.com/azure/azure-app-configuration/concept-config-file). Default value is **Default**.
+     - **Default**: Refers to the conventional configuration file schema widely adopted.
+     - **Kvset**: Refers to a [file schema](https://github.com/Azure/AppConfiguration/blob/main/docs/KVSet/KVSet.v1.0.0.schema.json) that contains all properties of an App Configuration key-value, including key, value, label, content type, and tags.
```

Comment:
Should we also indicate that the task Parameters  'Separator', 'Label', 'Content type', 'Prefix', 'Tags', and 'Depth' are not supported when using the Kvset profile or will it be confusing to indicate it here?

Reply:
Yes, I think that's a good idea.